patches/net/minecraft/world/item/alchemy/PotionBrewing.java.patch

```diff
@@ -56,11 +56,15 @@
              Optional<Holder<Potion>> optional = p_43531_.getOrDefault(DataComponents.POTION_CONTENTS, PotionContents.EMPTY).potion();
              if (optional.isEmpty()) {
                  return p_43531_;
-@@ -127,6 +_,7 @@
-     public static PotionBrewing bootstrap(FeatureFlagSet p_341301_) {
+@@ -124,9 +_,10 @@
+         }
+     }
+ 
+-    public static PotionBrewing bootstrap(FeatureFlagSet p_341301_) {
++    public static PotionBrewing bootstrap(FeatureFlagSet p_341301_, net.minecraft.core.RegistryAccess registryAccess) {
```

Comment:
The original method must be retained and marked as deprecated with a comment and the new one added as an overload. The original method can then call the new one with a default value like `RegistryAccess.EMPTY`.